front/src/App.vue

```diff
@@ -73,8 +73,10 @@ export default defineComponent({
     mounted() {
         this.fetchRooms();
         // polling
-        setInterval(() => {
+        this.socket.on("newMessage", () => {
```

Comment:
newMessage should be message, and the closure should take an argument.
```typescript
srv.socket.on("message", this.messages.push)
```

Reply:
fixes here [6dbe854](https://github.com/Draeggon/scratchy/pull/2/commits/6dbe8544adb2738a4f4a109bbf5f0cc90feaf8d6)